src/Compilers/CSharp/Portable/Symbols/Attributes/WellKnownAttributeData/ModuleEarlyWellKnownAttributeData.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    /// <summary>
+    /// Information decoded from early well-known custom attributes applied on a module.
+    /// </summary>
+    internal sealed class ModuleEarlyWellKnownAttributeData : EarlyWellKnownAttributeData
+    {
+        #region NonNullTypesAttribute
```

Comment:
Nit: why even bother with the region if it's the only thing in the whole file?

Reply:
See sibling files (they use same pattern regardless of count). This will make it easy to add the second one, when we need it.